athena-jdbc/README.md

```diff
@@ -54,6 +54,8 @@ Multiplexer provides a way to connect to multiple database instances of any type
 
 ```
 ${catalog}_connection_string    Database instance connection string. One of two types specified above. Required.
+                                If the catalog is myredshiftcatalog then the environment variable name should be myredshiftcatalog_connection_string
```

Comment:
adjust to "If the catalog *as registered with Athena* is myredshiftcatalog..."

Reply:
Done